app/controllers/trn_submissions_controller.rb

```diff
@@ -10,11 +10,11 @@ def create
       return render("trainees/check_details/show")
     end
 
-    trainee.submit_for_trn!
-
-    Dttp::RegisterForTrnJob.perform_later(trainee, current_user.dttp_id)
-    Dttp::RetrieveTrnJob.perform_with_default_delay(trainee)
+    if trainee.course_start_date <= Time.zone.today
```

Comment:
```suggestion
    if trainee.course_start_date.past?
```

Reply:
wooooooah 🤯

Reply:
oh wait but a course start date of 'today' should evaluate to true..

Reply:
We could use `unless course_start_date.future?` if that helps, but not very fussed, as it might be not readable enough. We probably need it to be uniform as well, so depending on the version we pick here, we might have to amend other occurences. cc: @avinhurry 
https://github.com/DFE-Digital/register-trainee-teachers/pull/1724/files#diff-c9a19baad4ab67f5ccb69f6a57a847bd3bd444ace7c3d8db25f7073486e6e651R98-R100